assets/src/block-validation/components/error/test/get-error-source-title.js

```diff
@@ -5,6 +5,7 @@ import { getErrorSourceTitle } from '../get-error-source-title';
 
 describe( 'getErrorSorceTitle', () => {
 	it( 'returns an empty string if nothing is passed', () => {
+		expect( getErrorSourceTitle( null ) ).toBe( '' );
```

Comment:
Does this need to be changed to:

```suggestion
		expect( getErrorSourceTitle( null ) ).toBe( 'Unknown' );
```

Reply:
Nevermind, my suggestion above seems to have been in error.